crates/nu-command/src/filesystem/cd.rs

```diff
@@ -169,11 +171,17 @@ impl Command for Cd {
         if let Some(oldpwd) = stack.get_env_var(engine_state, "PWD") {
             stack.add_env_var("OLDPWD".into(), oldpwd)
         }
-
         //FIXME: this only changes the current scope, but instead this environment variable
```

Comment:
These comments go with the add_env_var("PWD") so it may be better to move them to where it is now

Reply:
> These comments go with the add_env_var("PWD") so it may be better to move them to where it is now

I have moved them now